exporters/metric/cortex/cortex.go

```diff
@@ -157,28 +181,32 @@ func (e *Exporter) ConvertToTimeSeries(checkpointSet export.CheckpointSet) ([]*p
 	return timeSeries, nil
 }
 
+// createTimeSeries is a helper function to create a timeseries from a value and labels
+func createTimeSeries(record metric.Record, value apimetric.Number, extraLabels ...string) *prompb.TimeSeries {
```

Comment:
I see callers of this function sometimes passing `"__name__"` or `"name"`. Can you clarify either in this function documentation string or at the caller sites what the difference/nuance is if any?

Reply:
Thanks for catching that. They should all be __name__

Reply:
I'm assuming the underscores are to prevent clash with user defined "name" labels? Might be worth a clarification comment in places where this is occurring.

Reply:
It's required by Prometheus. I'll call that out.